VHbbAnalysis/Heppy/python/VHbbAnalyzer.py

```diff
@@ -178,26 +178,28 @@ def doHiggsHighPt(self,event) :
         event.H = event.hJets[0].p4()+event.hJets[1].p4()
 
 
-    def doHiggs3cj(self,event) :
-        event.H3cj = ROOT.reco.Particle.LorentzVector(0.,0.,0.,0.)
-        event.hJets3cj = []
-        event.aJets3cj = []
-	event.hjidx3cj = []         
-	event.ajidx3cj = []         
-        event.minDr3 = 0         
-        #3 jets interpretations, for the closest 3 central jets
-        if (len(event.jetsForHiggs) > 2 and event.jetsForHiggs[2] > 15.): 
-           aJetsForHiggs = [x for x in event.jetsForHiggs if x not in event.hJets]
-           j3=min(aJetsForHiggs, key = lambda x : min( deltaR( x.eta(), x.phi(), event.hJets[0].eta(), event.hJets[0].phi()),deltaR( x.eta(), x.phi(), event.hJets[1].eta(), event.hJets[1].phi() ) ) )
-           event.hJets3cj=event.hJets
-           event.hJets3cj.append(j3)
-#          event.hJets3cj=list(min(itertools.combinations(event.jetsForHiggs,3), key = lambda x : ( x[2]>15 and (deltaR( x[0].eta(), x[0].phi(), x[2].eta(), x[2].phi()) +  deltaR( x[1].eta(), x[1].phi(), x[2].eta(), x[2].phi()) ) ) ))
-           event.aJets3cj = [x for x in event.cleanJets if x not in event.hJets3cj]
-           event.hjidx3cj=[event.cleanJetsAll.index(x) for x in event.hJets3cj ]
-           event.ajidx3cj=[event.cleanJetsAll.index(x) for x in event.aJets3cj ]
-           event.aJets3cj+=event.cleanJetsFwd
-           event.H3cj = event.hJets3cj[0].p4()+event.hJets3cj[1].p4()+event.hJets3cj[2].p4()
-           event.minDr3 = min(deltaR( event.hJets3cj[0].eta(), event.hJets3cj[0].phi(), event.hJets3cj[2].eta(), event.hJets3cj[2].phi()) ,  deltaR( event.hJets3cj[1].eta(), event.hJets3cj[1].phi(), event.hJets3cj[2].eta(), event.hJets3cj[2].phi()))  
+    def doHiggsAddJetsdR08(self,event) :
+        event.hJetsaddJetsdR08 = [x for x in event.hJetsCSV]
```

Comment:
can you explain what is this meant for? why do you need a copy of all these collections?!